### ISSUE TITLE
Docstring of load_training_state says it returns a dict but it actually returns a list

### ISSUE THREAD

**Issue by white-alistair** [neuraloperator/neuraloperator#478]
https://github.com/neuraloperator/neuraloperator/blob/202426c28fc33dc7ac61511eb9cc9cba6a8e3cbd/neuralop/training/training_state.py#L46

https://github.com/neuraloperator/neuraloperator/blob/202426c28fc33dc7ac61511eb9cc9cba6a8e3cbd/neuralop/training/training_state.py#L98

**Comment by white-alistair** [comment]
Of course. Safe to assume the implemented behaviour is correct and it's just the docstring that needs correcting?

**Comment by JeanKossaifi** [comment]
Yes - though a dict could be clearer - what do you think @dhpitt?

**Comment by dhpitt** [comment]
Currently the docstring matches the implementation after the fixes to training state for 1.0.1. Either could work, what do you think?
